modules/core/arrow-core-data/src/test/kotlin/arrow/core/SequenceKTest.kt

```diff
@@ -52,6 +56,11 @@ class SequenceKTest : UnitSpec() {
     val EQ = SequenceK.eq(Int.eq())
 
     testLaws(
+      MonadPlusLaws.laws(
```

Comment:
What is the point testing `MonadPlus` and `MonadLogic` seperately? `MonadLogic` alone should test both right?